frontend/src/app/App.tsx

```diff
@@ -28,7 +28,7 @@ import { AccessReviewProvider } from '~/concepts/userSSAR';
 import { ExtensibilityContextProvider } from '~/plugins/ExtensibilityContext';
 import useFetchDscStatus from '~/concepts/areas/useFetchDscStatus';
 import { OdhPlatformType } from '~/types';
-import useDevFeatureFlags from './useDevFeatureFlags';
+import useDevFeatureFlags from './FeatureFlags/useDevFeatureFlags';
```

Comment:
Please follow the camel-case convention of the folder name, so here the folder name should be `featureFlags` starting with a lower-case `f`.